src/services/eip/format.ts

```diff
@@ -41,7 +41,7 @@ export default ({
     accountId: account,
     arn: eipAllocationArn({region, account, id}),
     region,
-    vpc: domain === 'vpc' ? t.yes : t.no,
+    isVpc: domain === 'vpc' ? t.yes : t.no,
```

Comment:
Any reason for this change? Or what would be a good name for this?

Reply:
I rename it because was needed to change the current "vpcs" field connection to vpc and after that both are overlapped.

![image](https://user-images.githubusercontent.com/64504276/163264350-29768c9e-06be-4de0-a805-d15ece7f0798.png)

But well, I can rename it again if that is not the most appropriated. What do you think?

Reply:
Makes sense to rename it, I just wonder what the best naming for this property would be. @ckoning any ideas?

Reply:
`isVpc` is fine for the computed value